riscv/insns/vfunary0_vv.h

```diff
@@ -0,0 +1,110 @@
+// VFUNARY0 encoding space
```

Comment:
This should be expanded out into 5 files, one per instruction.  I've made the corresponding change to riscv-opcodes: https://github.com/riscv/riscv-opcodes/commit/0c06e7381689977480fd9ec6a1711f8eb48c6382#diff-b8de9572fc6a9a303c3300f15cf8199c

Reply:
OK

Reply:
@chihminchao

Reply:
I will follow change in  riscv-opcodes.

Reply:
also fix vfclass.v opcode and has included it in my previous PR.
https://github.com/riscv/riscv-opcodes/pull/29